gradle/wrapper/gradle-wrapper.properties

```diff
@@ -16,6 +16,6 @@
 
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.5.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.1.1-all.zip
```

Comment:
Please do not upgrade this file manually, but run the `./gradlew wrapper` task, so that also the binary files will be upgraded (see the commit I'm about to push)